kong/templates/nginx_kong_stream.lua

```diff
@@ -119,5 +119,15 @@ server {
         Kong.log()
     }
 }
+
+server {
```

Comment:
We should probably only inject this block when `strategy` is `off` (or whenever the `/config` endpoint is available anyway)? Especially given NGINX doesn't handle unix socket file very elegantly in some cases...